support.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
@lilin90 PTAL.
I'll update the description in all other docs and link to this file later.